log/logrus/logrus_logger.go

```diff
@@ -0,0 +1,31 @@
+package log
+
+import (
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/sirupsen/logrus"
+)
+
+type logrusLogger struct {
+	*logrus.Logger
+}
+
+// NewLogrusLogger takes a *logrus.Logger and returns
+//a logger that stisfies the go-kit log.Logger interface
```

Comment:
- Missing a space after the `//` and a period (.) at the end of the sentence.
- s/stisfies/satisfies